localstack/services/awslambda/lambda_api.py

```diff
@@ -65,7 +65,13 @@
 LAMBDA_ZIP_FILE_NAME = 'original_lambda_archive.zip'
 LAMBDA_JAR_FILE_NAME = 'original_lambda_archive.jar'
 
-DEFAULT_BATCH_SIZE = 10
+INVALID_PARAMETER_VALUE_EXCEPTION = 'InvalidParameterValueException'
+
+BATCH_SIZE_MAP = {
```

Comment:
nit: I'd probably rename this to `BATCH_SIZE_RANGES`, to use a slightly more descriptive name.